static/js/message_edit.js

```diff
@@ -976,13 +976,14 @@ export function delete_topic(stream_id, topic_name, failures = 0) {
         data: {
             topic_name,
         },
-        success() {},
-        error(xhr) {
```

Comment:
Do we want to do any retries on _other_ failures?  "Not really" seems like the most likely answer, but seeing the `error` handler disappear makes me think about it a little.

Reply:
Hmm, good question, but I don't see any other errors that should mean a retry :thinking:  The `error` handler didn't exist before #22988, it was just for the timeout scenario.

Reply:
Yeah, in retrospect this is more of a generalized musing about needing to think about how and when we do retries in the client.  I think this is fine for now as-is, without an explicit error handling block.